src/runtime_src/xdp/profile/device/device_intf.cpp

```diff
@@ -605,6 +605,7 @@ DeviceIntf::~DeviceIntf()
             case AXI_NOC :               nocList.push_back(new NOC(mDevice, i, &(map->m_debug_ip_data[i])));
                                          break;
             default : break;
+            // case ACCEL_DEADLOCK_DETECTOR: break;
```

Comment:
Hmm...I would have expected the code to look like this:

```
case ACCEL_DEADLOCK_DETECTOR:
case AXI_STREAM_PROTOCOL_CHECKER:
default:
  break;
```

Why the difference?